### PR TITLE
add exp name in text log

### PR DESCRIPTION
**Pull request by nbei**
@hellock

### CODE REVIEW

File: mmcv/runner/hooks/logger/text.py [hunk 1]
```diff
@@ -36,6 +41,13 @@ def _get_max_memory(self, runner):
         return mem_mb.item()
 
     def _log_info(self, log_dict, runner):
+        if runner.meta is not None and 'exp_name' in runner.meta:
+            if (self.every_n_inner_iters(
+                    runner,
+                    self.interval_exp_name)) or self.end_of_epoch(runner):
+                exp_info = 'Exp name: {}\t'.format(runner.meta['exp_name'])
```
Comment (yhcao6) [inline mmcv/runner/hooks/logger/text.py:48]: Use f-string instead format.

Comment (nbei) [reply mmcv/runner/hooks/logger/text.py:48]: I guess python3.5 may not support f-string?

File: mmcv/runner/hooks/logger/text.py [hunk 2]
```diff
@@ -14,9 +14,14 @@
 @HOOKS.register_module
 class TextLoggerHook(LoggerHook):
```
Comment (hellock) [inline mmcv/runner/hooks/logger/text.py:15]: Add a docstring.

File: mmcv/runner/hooks/logger/text.py [hunk 3]
```diff
@@ -36,6 +41,13 @@ def _get_max_memory(self, runner):
         return mem_mb.item()
 
     def _log_info(self, log_dict, runner):
+        if runner.meta is not None and 'exp_name' in runner.meta:
+            if (self.every_n_inner_iters(
+                    runner,
+                    self.interval_exp_name)) or self.end_of_epoch(runner):
+                exp_info = f"Exp name: {runner.meta['exp_name']}\t"
```
Comment (hellock) [inline mmcv/runner/hooks/logger/text.py:48]: `\t` may not be necessary since it is a single line.

**Pull request merged**